Assets/Scripts/UI/MainMenu/MainMenu.cs

```diff
@@ -0,0 +1,46 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+
+using UnityEditor;
```

Comment:
UnityEditor is stripped in builds, so building will fail if non-editor code uses it.